### PR TITLE
feat: add addWebpackPlugin

### PR DESCRIPTION
**Pull request by zhigang1992**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -150,6 +150,10 @@ Adds the provided alias info into webpack's alias section. Pass an object litera
 
 Adds the provided resolve info into webpack's resolve section. Pass an object literal with as many entries as you'd like, and the whole object will be merged in.
 
+### addWebpackPlugin(plugin)
+
+Adds the provided plugin info into webpack's plugin array. Pass an plugin defined with  `new webpack.DefinePlugin({...})`
```
Comment (FezVrasta) [inline readme.md]: "a plugin"

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "customize-cra",
-  "version": "0.2.12",
+  "name": "@zhigang1992/customize-cra",
```
Comment (FezVrasta) [inline package.json]: Please don't edit the package.json, we'll do that when we release a new version

Comment (zhigang1992) [reply package.json]: sry... that wasn't mean to be included.

**Comment by kumarharsh** [comment]
It seems to me that I'll need to install webpack as a dev dependency just to use plugins. Am I correct?

**Comment by kurnal** [comment]
@kumarharsh have you been able to figure out how to add the webpack plugin. no matter what I do it does not append anything into ENV for me and it can't set any global variables.

**Comment by kumarharsh** [comment]
Sorry, I don't remember much (moved on from the project). I did manage to figure it out, but don't remember the specifics at all :)

**Pull request merged**